code/modules/mob/mob.dm

```diff
@@ -1697,6 +1709,30 @@ Use this proc preferably at the end of an equipment loadout
 			src.callOnFace -= .
 
 
+/mob/proc/StartMoving()
```

Comment:
Please explain the reasons of these wrappers in a comment above so that future coders can understand why those procs exists at a glance.

Reply:
added a comment the line before